keyboards/handwired/dactyl_left/keymaps/default/layers.json

```diff
@@ -0,0 +1 @@
+[["KC_EQL", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_DEL", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_BSPC", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_LSFT", "LCTL_T(KC_Z)", "KC_LSFT", "KC_COMM", "KC_DOT", "KC_B", "LT(1,KC_GRV)", "LCTL(KC_Z)", "LALT(KC_LSFT)", "KC_LEFT", "KC_RGHT", "LALT_T(KC_APP)", "KC_LGUI", "KC_HOME", "KC_LALT", "KC_SPC", "LT(1,MO(layer))", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_BSLS", "KC_H", "KC_J", "KC_K", "KC_L", "LT(2,KC_SCLN)", "LGUI_T(KC_QUOT)", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "LCTL_T(KC_SLSH)", "KC_RSFT", "KC_UP", "KC_DOWN", "KC_LBRC", "KC_RBRC", "TT(1)", "KC_RALT", "LCTL_T(KC_ESC)", "KC_PGUP", "KC_PGDN", "KC_TAB", "KC_ENT"], ["KC_TRNS", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_TRNS", "KC_EXLM", "KC_AT", "KC_LCBR", "KC_RCBR", "KC_PIPE", "KC_TRNS", "KC_HASH", "KC_DLR", "KC_LPRN", "KC_RPRN", "KC_GRV", "KC_TRNS", "KC_PERC", "KC_CIRC", "KC_LBRC", "KC_RBRC", "KC_TILD", "RESET", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_UP", "KC_7", "KC_8", "KC_9", "KC_ASTR", "KC_F12", "KC_DOWN", "KC_4", "KC_5", "KC_6", "KC_PLUS", "KC_TRNS", "KC_AMPR", "KC_1", "KC_2", "KC_3", "KC_BSLS", "KC_TRNS", "KC_TRNS", "KC_DOT", "KC_0", "KC_EQL", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS"], ["KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MS_U", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MS_L", "KC_MS_D", "KC_MS_R", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_BTN1", "KC_BTN2", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_MPLY", "KC_TRNS", "KC_TRNS", "KC_MPRV", "KC_MNXT", "KC_TRNS", "KC_TRNS", "KC_VOLU", "KC_VOLD", "KC_MUTE", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_TRNS", "KC_WBAK"]]
```

Comment:
This file doesn't match your hardware. It should be removed from the pull request.

Reply:
Updated! Thank you!